setup.py

```diff
@@ -42,9 +42,11 @@
     include_package_data=True,
```

Comment:
The hover-over text for copying code snippets overlaps with transparency if the code snippet is too long:

![Screenshot from 2021-07-16 12-19-47](https://user-images.githubusercontent.com/16181459/125979366-c1c5c8f8-dce1-4bc6-ab54-a72f59324e97.png)

Could we have an opaque background for the "Copy" text?

Reply:
I think so! I'll investigate 🔎

Reply:
Changed here: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3355/commits/452a37076055c0c931c163b967a2dafc4618679b

Reply:
Perfect! :tada: